socketshark/receiver.py

```diff
@@ -113,7 +108,7 @@ async def _reader(self, connection, once=False):
         while True:
```

Comment:
(Commenting here since I cannot comment above)

Maybe let's call it `_reader_for_connection`?